react-native/App.tsx

```diff
@@ -0,0 +1,40 @@
+import React, {useState} from 'react';
+
+import {NavigationContainer} from '@react-navigation/native';
+import {createStackNavigator} from '@react-navigation/stack';
+
+import Home from './components/Home';
+import Meeting from './components/Meeting';
+
+const RootStack = createStackNavigator();
+
+const App = () => {
+  const [text, onChangeText] = useState('ThisRoomNameIsATestForYouToTest');
```

Comment:
Don't use any room text by default.

Reply:
Rather than calling this text let's call it room.

Reply:
Also, shouldn't this be paaart of the Home screen, which passes the room as a parameter to the Meeting route?

Reply:
In the first place I had the whole state management in the app component. At the end I changed/removed some stuff and I forget about this. Will do!

Reply:
Yes, but there are 2 problems with it:

1- The namig is bad, "text" means nothing, it's the room name
2- It's in the wrong place. The App should know nothing about it. The text field is in the home screen, which can get it and then send it as a route parameter when navigating to the Meeting screen.

Reply:
Ah yes. Forgot about the route params.